about.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>About | WBS 2022</title>
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+  <link rel="stylesheet" href="css/styles.css">
+  <script src="https://kit.fontawesome.com/4f5822c69a.js" crossorigin="anonymous"></script>
+</head>
+<body>
+  <div class="container-fluid p-0 background-header border-bottom">
+    <header id="navbar">
+      <div class="container-fluid p-0 d-none d-md-block top-navbar">
+        <div class="container-md p-0">
+          <nav class="nav justify-content-end">
+            <a href="#" class="nav-item nav-link a"><i class="fab fa-facebook-f"></i></a>
+            <a href="#" class="nav-item nav-link a"><i class="fab fa-twitter"></i></a>
+            <a href="#" class="nav-item nav-link a">English</a>
+            <a href="#" class="nav-item nav-link a">My page</a>
+            <a href="#" class="nav-item nav-link a">Logout</a>
+          </nav>
+        </div>
+      </div>
+      <div class="container-fluid p-0 navbar-bg shadow-nav">
+        <div class="container-md navbar navbar-expand-md navbar-light">
+          <button type="button" class="navbar-toggler border-0" data-bs-toggle="offcanvas" data-bs-target="#navbarSlide">
+            <span class="navbar-toggler-icon"></span>
+          </button>
+          <nav class="offcanvas offcanvas-start navbar-collapse border-0" tabindex="-1" id="navbarSlide">
+            <div class="offcanvas-header align-self-stretch  me-md-auto p-md-0 logo-container">
+              <div class="offcanvas-title">
+                <a href="./index.html">
+                  <img src="images/WBS-logo.png" alt="wbs logo" class="logo">
+                </a>
```

Comment:
There is the link to the homepage

Reply:
@helio3197,
According to the project requirement: If you are on the `Home page` there should be a `nav Link` to `About page` and if you are on the `About page` there should be a `nav Link` to `Home page`. You can add `Home` to your `nav bar links`

Reply:
Thank you for your feedback, I understand now.